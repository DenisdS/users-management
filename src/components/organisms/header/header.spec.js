import renderer from 'react-test-renderer';
import React from 'react';
import Header from './header'

it('Verify if header renders correctly', () => {
  const tree = renderer
    .create(
      <Header>
        <img src="" alt=""/>
      </Header>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
