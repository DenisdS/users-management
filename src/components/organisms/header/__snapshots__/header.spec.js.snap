// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify if header renders correctly 1`] = `
<header>
  <img
    alt=""
    src=""
  />
</header>
`;
